v0.3/handbook/libraries/stdlib.md

```diff
@@ -416,6 +416,62 @@ commit
 <table class="libitem">
   <tr>
     <td colspan="2">
+      <b>math/pow</b> - Raise a value to an exponent
+    </td>
+  </tr>
+
+  <tr>
+    <td>
+      <ul>
+        <li><strong>value</strong> - the mantissa</li>
```

Comment:
Is mantissa the right term here? Even if it is, we might be better off just saying number instead so as to reduce confusion.